int-set/src/lib.rs

```diff
@@ -189,6 +190,16 @@ impl<T: Domain<T>> IntSet<T> {
         }
     }
 
+    /// Returns the smallest member of this set if there is one.
+    pub fn min(&self) -> Option<T> {
```

Comment:
`BTreeSet` calls these fns `first` and `last`, maybe that would make sense? See https://doc.rust-lang.org/1.78.0/std/collections/struct.BTreeSet.html#method.last

Reply:
sounds good.